tool/python/examples/mnist_rbm1.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__),'..')) 
+from singa.model import *
+from singa.datasets import rbm 
+
+rbmid = 1
+X_train, X_test, workspace = rbm.load_data(rbmid)
+
+m = Sequential('rbm'+str(rbmid), sys.argv)
```

Comment:
Ok. Or call it energy model?

Sent from my iPhone

> On 18 Dec 2015, at 5:37 PM, chonho <notifications@github.com> wrote:
> 
> In tool/python/examples/mnist_rbm1.py:
> 
> > @@ -0,0 +1,21 @@
> > +#!/usr/bin/env python
> > +import sys, os
> > +sys.path.append(os.path.join(os.path.dirname(__file__),'..')) 
> > +from singa.model import *
> > +from singa.datasets import rbm 
> > +
> > +rbmid = 1
> > +X_train, X_test, workspace = rbm.load_data(rbmid)
> > +
> > +m = Sequential('rbm'+str(rbmid), sys.argv)
> instead of Sequential, Can I call/create "Bidirectional", inherited from Model class?
> 
> —
> Reply to this email directly or view it on GitHub.
>